index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="cards">
+      <div class="card">
+        <div class="card__content">
+          <img
+            src="images/Snapshoot-Portfolio.png"
+            alt=""
+            class="card__image"
+          />
+          <h2>Multi-Post Stories</h2>
+          <div class="card__tag">
+            <div class="tag">
+              <p>Canopy</p>
+            </div>
+            <div class="tag__inactive">
+              <img src="images/Counter.png" alt="" />
+              <p>Back End Dev</p>
+            </div>
+            <div class="tag__inactive">
+              <img src="images/Counter.png" alt="" />
+              <p>2015</p>
+            </div>
+          </div>
+          <div class="card__description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="card__btn">
+            <div class="mini__btn">
+              <div class="btn">
+                <p>html</p>
+              </div>
+              <div class="btn">
+                <p>css</p>
+              </div>
+              <div class="btn">
+                <p>javascript</p>
+              </div>
```

Comment:
- [ ]  It will be nice for you to ensure that the list of languages uses a `ul` tag to group them, and `li` per language.
- [ ] Kindly try to change that. And do it for all projects. 😃